test/unit/inet-address-tests.js

```diff
@@ -168,28 +168,44 @@ describe("InetAddress", function () {
             ].forEach(function (address) {
                 assert.throws(function () {
                     InetAddress.fromString(address);
-                }, TypeError);
+                }, Error);
             });
         });
         it("should throw TypeError when the string is not a valid address", function () {
             assert.throws(function () {
                 InetAddress.fromString("127.0.0.1.10");
-            }, TypeError);
+            }, Error);
             assert.throws(function () {
                 InetAddress.fromString("127.0.");
-            }, TypeError);
+            }, Error);
             assert.throws(function () {
                 InetAddress.fromString(" 127.10.0.1");
-            }, TypeError);
+            }, Error);
             assert.throws(function () {
                 InetAddress.fromString("z");
-            }, TypeError);
+            }, Error);
             assert.throws(function () {
                 InetAddress.fromString("::Z:11:2233:4455:aa:bb");
-            }, TypeError);
+            }, Error);
             assert.throws(function () {
                 InetAddress.fromString(" ::a:11");
-            }, TypeError);
+            }, Error);
+        });
+    });
+    describe("setters", function () {
+        it("should throw errors when trying to set a value", function () {
+            const val = new InetAddress(
+                utils.allocBufferFromArray([127, 0, 0, 1]),
+            );
+            assert.throws(function () {
+                val.buffer = utils.allocBufferFromArray([127, 0, 0, 1]);
+            }, Error);
+            assert.throws(function () {
+                val.version = 6;
+            }, Error);
+            assert.throws(function () {
+                val.length = 4;
+            }, Error);
```

Comment:
Please check is the correct errors -> `name` and `message`.

Reply:
I don't think it is necessary as in first test written by DataStax  error names are not checked. Also in case of changing error message, tests also would need to be changed, slowing down development time.